README.md

```diff
@@ -55,6 +55,14 @@ execute commands so ensure *posix-spawn* gem is installed
 ``gem install posix-spawn``
 
 
+## Expedited Installation
+
+The .travis file contains the necessary sh files to install the needed libraries. Simply execute the following commands:
```

Comment:
We can avoid the mention of travis file here. This section could be better placed under the **System Requirements** heading, with the following installation details:

```
  $ sh .travis/.install_multiple_scms.sh
  $ sh .travis/.install_dependencies.sh
  $ gem install posix-spawn
  $ gem install nokogiri -v 1.8.1
```
We can remove the existing posix-spawn installation information in **System Requirements**.

Reply:
A better fix will be to have the following structure

```
ohloh_scm/
  setup/
    install_multiple_scms.sh
    install_dependencies.sh
    install.sh
      source ./install_multiple_scms.sh
      source ./install_dependencies.sh
      gem install posix-spawn
      gem install nokogiri -v 1.8.1
  .travis.yml
    ...
    sh setup/install.sh
```